packages/io/x3d-serializer/src/index.js

```diff
@@ -180,14 +201,20 @@ const convertAppearance = (object, colorField, options) => {
   const colorRGB = object.color.slice(0, 3)
   const color = colorRGB.join(' ')
   const transparency = roundToDecimals(1.0 - object.color[3], options)
-  return ['Appearance', ['Material', { [colorField]: color, transparency }]]
+  const materialFields = { [colorField]: color, transparency }
+  if (colorField === 'diffuseColor') {
+    Object.assign(
+      materialFields,
+      { specularColor: '0.2 0.2 0.2', shininess: options.shininess })
```

Comment:
Can something similar be done here for material attributes?

Reply:
Ok, we can look for all Material attributes as outlined above.

Reply:
Cool.

By the way, what ever becomes supported in the serializer should be supported in the deserializer. We should support the round trip...

Reply:
I noticed that the deserializer uses obect.id for DEF names. Let me then use .id here as well
 rather than .name

Reply:
Would material attributes in a sub property be appropriate: object.material.ambientIntensity ?

Reply:
That seems reasonable. Please add some notes to the README file.